tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -23,6 +23,9 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
   val dynsigs       = mutable.Set.empty[Sig]
   val dynimpls      = mutable.Set.empty[Global]
 
+  private case class DeleyedMethod(owner: Global.Top, sig: Sig, pos: Position)
```

Comment:
This, and associated use sites.
```suggestion
  private case class DelayedMethod(owner: Global.Top, sig: Sig, pos: Position)
```